src/Raven.Server/Config/Categories/ReplicationConfiguration.cs

```diff
@@ -6,7 +6,14 @@ namespace Raven.Server.Config.Categories
 {
     public class ReplicationConfiguration : ConfigurationCategory
     {
-        [DefaultValue(600)]
+		[Description("Threshold under which a connection is considered active")]
+		[DefaultValue(30)]
+		[TimeUnit(TimeUnit.Seconds)]
+		[ConfigurationEntry("Raven/Replication/ActiveConnectionTimeoutInSec")]
+		[ConfigurationEntry("Raven/Replication/ActiveConnectionTimeout")]
```

Comment:
similar to
  [DefaultValue(600)]
        [TimeUnit(TimeUnit.Seconds)]
        [ConfigurationEntry("Raven/Replication/IndexAndTransformerReplicationLatencyInSec")]
        [ConfigurationEntry("Raven/Replication/IndexAndTransformerReplicationLatency")]
        public TimeSetting IndexAndTransformerReplicationLatency { get; set; }

        [Description("Number of seconds after which replication will stop reading documents from disk")]
        [DefaultValue(30)]
        [TimeUnit(TimeUnit.Seconds)]
        [ConfigurationEntry("Raven/Replication/FetchingFromDiskTimeoutInSec")]
        [ConfigurationEntry("Raven/Replication/FetchingFromDiskTimeout")]
        public TimeSetting FetchingFromDiskTimeoutInSeconds { get; set; }